Schedule sneak peek and Threema sponsor mention (#70)
RustFestEU/blog.rustfest.eu: _posts/2018-10-26-schedule-sneak-peek-rome.md

---
layout: post
title: "Schedule sneak peek"
socialImageSrc: "/assets/posts/rome/twirf-rustbridge.jpg"
authors:
- dns2utf8
---

The schedule is long awaited and it is the one thing the organisation has to make sure is interesting and digestible.
Plus the topics should be varied.

So where is the schedule? Since the talks are already [published on the website](https://rome.rustfest.eu/talks/), how hard can it be?
Well here is the current draft:

| Time  | Talk or Event      |
|-------|--------------------|
| 08:30 | Registration opens |
| 09:00 | Introduction from local orga team + sponsors |
| 09:30 | Opening keynote |
| ...   | ... |
| 10:45 | Coffee break |
| ...   | ... |
| 12:30 | Lunch break |
| 14:00 | Afternoon talks |
| ...   | ... |
| 15:05 | Coffee break |
| ...   | ... |
| 16:55 | Closing keynote |
| 17:40 | Tips and final words from the team |
| 17:55 | Family photo |
| 18:00 | Doors close |

You might be wondering "did they just write times like `:40`?"
Yes we did!

> There are some Swiss people in the orga and to maximize the amount of talks we will be punctual and on time.

Reviewer: Thanks to the Swiss people for keeping at bay the Italians :-))
(joking)

Author: Hope people get the wink without an emojy ;)

If you are a speaker, we will give you hints on when to come to an end.
Also, if you would like to allow the audience to ask questions, that has to fit into you slot.

> If you are an attendee, the talks will start really on time and if you get the microphon ask a querstion, ask one short question each (no opinions).

Reviewer: ```suggestion
If you are an attendee, please be aware that talks will start definitively on time and - unless they finish ahead of the allotted time slot - there will be time only for a quick question or two.
```

> The speakers will be around after their talks where you can discuss everything including opinions.

Reviewer: ```suggestion
Speakers will be around after their talks for in depth discussions.
```

# We still sell tickets

Our final batch of tickets is 🎟 [live on tito](https://ti.to/asquera-event-ug/rustfest-rome-2018/) 🎟.

These tickets will stay around until they sell out or shortly before the conference.

# Our Friends at Threema

> We’re happy to welcome Threema as a sponsor for Paris.

Reviewer: Rome :-)

Author: Ahhh, ... so close '^^
